CHANGES.md

```diff
@@ -1,12 +1,18 @@
 # legacysurvey Change Log
 
-## 9.2.0 (DR9, unreleased)
-
-- *Planned*: correct units for `ccdskycounts`
+## 10.0.0 (DR10, unreleased)
+
+- Near-final updates for DR10
+  ([PR#165](https://github.com/legacysurvey/legacysurvey/pull/165)):
+    - Includes a sufficient set of DR10 pages for an initial release.
+    - Still missing details of the syntax for the cutout server.
+    - Still missing details about the DR10 sky pattern correction.
+    - Addresses [issue #160](https://github.com/legacysurvey/legacysurvey/issues/160).
+- *Planned for DR9*: correct units for `ccdskycounts`
```

Comment:
Does this correspond to the changes in units `erg...` -> `erg/s...` that I saw in many of the files?  If so we should remove the "Planned for DR9".  If not, then this should not be included  in the changes for DR10.

Reply:
No, this is a different issue. In any case, I removed all of the "Planned for DR9" comments.